test/model/avatar_url_test.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:checks/checks.dart';
+import 'package:test/scaffolding.dart';
+import 'package:zulip/model/avatar_url.dart';
+
+class ExampleUser {
+  final Uri realm;
+  final int userId;
+  final String email;
+
+  ExampleUser({
+    required this.realm,
+    required this.userId,
+    required this.email,
+  });
+}
+
+void main() {
+  var egUser = ExampleUser(
+    realm: Uri.parse("https://zulip.example.org"),
+    userId: 1417,
+    email: "sample@zulip.org",
+  );
+
+  double defaultSz = 30;
```

Comment:
"size", not "sz": https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations

Reply:
[ecec588](https://github.com/zulip/zulip-flutter/pull/534/commits/ecec58836637ac686e88dade6d241dbd31bcbefc)